data/remnant/remnant missions.txt

```diff
@@ -1060,6 +1060,334 @@ mission "Remnant: Learn Sign Follow Up"
 			`	"Congratulations, Captain <last>!" the director signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways than just by blood. I look forward to seeing what results from your work with us." The director is interrupted by the arrival of other Remnant asking for something and bids you farewell with an apologetic gesture.`
 				decline
 
+	
+mission "Remnant Rescue"
+	job
+	repeat
+	name "Search and Rescue: Scout"
+	description "The <npc> has gone missing during a scouting mission in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 30
+	npc accompany save
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+			not system "Edusa"
+		personality derelict timid plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant
+				Starling
+			variant
+				Gull
+	on complete
+		"rescue missions" ++
+		payment 200000
+		dialog
+			`Your communicator notifies you of a resource allocation of <payment>.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+
+# The Rescue Dummy is meant to suicide as soon as possible on entering the randomized system.
+ship "Rescue Dummy"
+	#sprite "ship/behemoth"
```

Comment:
The reason why it is commented out is because you're not meant to see it. You're meant to find the wreckage, not watch it explode (see: dialogue).

As for why it isn't a Remnant ship, I was having trouble quickly discerning the black ships on the black background when I was testing this, so I used a large, light ship for visibility purposes (I could've used a Sojourner, too). Blame my poor eyesight.